mysql-test/main/repair_symlink-5543.result

```diff
@@ -1,7 +1,7 @@
 create table t1 (a int) engine=myisam data directory='MYSQL_TMP_DIR';
 insert t1 values (1);
 # Some systems fail with errcode 31 (FreeBSD), 40, or 90 (MIPS) when doing openat,
-# while others don't have openat and fail with errcode 20.
+# while others don't have openat and fail with errcode 20. MacOS has 62.
```

Comment:
IMHO it might be clearer to say "MacOS fails with 62"

Reply:
Ack, Will correct. multiauth still failing, looks like the output line has exceeded the mysqltest length and the place I changed this isn't correct.

Reply:
@grooverdan I have a fix for the `multiauth` failure (patch will be for `libmariadb` module).  Are you still planning to make the change above and update this PR?  Thanks.